### PR TITLE
docs(pagination): Add demo for disabled and active states

### LINKED ISSUE #3688: docs(pagination): Add demo for disabled and active states
related to #3237

### PR DESCRIPTION
**Pull request by artolshansky**
Demo for switching disable state

close #3688

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+pagination/demos/index.ts [hunk 1]
```diff
@@ -2,10 +2,12 @@ import { DemoPaginationBasicComponent } from './basic/basic';
 import { DemoPaginationPagerComponent } from './pager/pager';
 import { DemoPaginationLimitComponent } from './limit/limit';
 import { DemoPaginationStylingComponent } from './styling-global/styling-global';
+import { DemoPaginationStatesComponent } from './states/states';
```
Comment (IlyaSurmay) [inline demo/src/app/components/+pagination/demos/index.ts]: `states` says nothing about disabled state. `DemoPaginationDisabledComponent` would be better

File: demo/src/app/components/+pagination/demos/states/states.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-pagination-states',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+pagination/demos/states/states.ts]: same

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+pagination/pagination-section.list.ts [hunk 3]
```diff
@@ -35,6 +36,13 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/basic/basic.html'),
         outlet: DemoPaginationBasicComponent
       },
+      {
+        title: 'States',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+pagination/pagination-section.list.ts]: `states` - > `disabled`

File: demo/src/app/components/+pagination/pagination-section.list.ts [hunk 4]
```diff
@@ -35,6 +36,13 @@ export const demoComponentContent: ContentSection[] = [
...
+        anchor: 'pagination-states',
+        component: require('!!raw-loader?lang=typescript!./demos/states/states.ts'),
```
Comment (IlyaSurmay) [inline demo/src/app/components/+pagination/pagination-section.list.ts]: folder names as well

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
For bootstrap 3 all buttons under pagination flows right (as on screenshot)
How it looks for bs3:
bs3togglerbutton

Otherwise looks good

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — DISMISSED** [review]

**Pull request merged**